### PR TITLE
Squids but i think they work this time

### PR DESCRIPTION
**Pull request by Wilchenx**
<buy advertisement here>

Hopefully fixes lack of sprites

### CODE REVIEW

File: config/example/admins.txt [hunk 1]
```diff
@@ -135,3 +135,4 @@ Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
 NicBR = Game Master
+Wilchen = Host
```
Comment (tsaoToast) [inline config/example/admins.txt:138]: speedmerge

Comment (Wilchenx) [reply config/example/admins.txt:138]: configs are serverside and this is merely so i can test this on my end

Comment (tsaoToast) [reply config/example/admins.txt:138]: just makin' the joke, don't worry. did you figure out what the issue was last time?

Comment (CthulhuOnIce) [reply config/example/admins.txt:138]: ok

**Pull request merged**